tutorials/en_us/JARS/ADVANCED.md

```diff
@@ -2,45 +2,67 @@
-  and the server and will refuse connection if there are mod incompatibilities. Dedicating more ram is recommended or even having a dedicated machine to running a
-  modded server is recommended.
-- [Fabric](https://fabricmc.net/) - Faster than Forge, doesn't run Forge mods and has lesser mod support
+- [Forge](http://files.minecraftforge.net/) - Forge is a server software based on the MCP. Both the server and client need to be running the mods.
 
-## Most popular MOD + PLUGINS server JARs:
+- [Fabric](https://fabricmc.net/) - Fabric, unlike Forge, is not based on the MCP. This allows it to be updated for snapshots easier. It is incompatible with Forge.
 
-[Sponge](https://www.spongepowered.org) - Enables use of mods and plugins which are made for Sponge, makes your server extra spicy if you know how to do it right!
-ore.spongepowered.org | The place to get the plugins (has LuckPerms :O)
+### Plugins + Mods:
+
+- [Magma](https://magmafoundation.org/) - Minecraft Forge Hybrid server implementing the Spigot/Bukkit API (Cauldron for 1.12)
+
+- [Mohist](https://mohistmc.com/) - Minecraft Forge Hybrid server implementing the Paper/Spigot/Bukkit API(1.12.2/1.16), formerly known as Thermos/Kettle/Cauldron/MCPC+
+
+- [Arclight](https://github.com/IzzelAliz/Arclight) - A Minecraft Bukkit(1.15/1.16) server implementation on Forge using Mixin
+
+- [SpongeForge](https://www.spongepowered.org/downloads/spongeforge) - SpongeForge allows both SpongeVanila plugins as well as Forge mods on one server.
+
+- [Cardboard](https://www.curseforge.com/minecraft/mc-mods/cardboard) - Cardboard (formerly Bukkit4Fabric) is a Fabric mod that adds support for the popular Bukkit/Spigot modding API. This mod lets you use plugins that are made for Bukkit and it's derivatives (Spigot & Paper) with Fabric. It's not a server software but allows fabric servers to run bukkit plugins.
+
+## Network Software:
+
+- [BungeeCord](https://www.spigotmc.org/wiki/bungeecord/) - Base for the Waterfall fork, these JARs are used for configuring networks of which an example is Hypixel or Mineplex, it is used for distributing workload and players to different lobbies (Different server instances run with separate files and data).
+  No, this can't be used as a normal JAR and requires at least 3 servers/instances, one is BungeeCord (or it's fork) which is called the proxy, players connect to this proxy and are sent to the default server which is set in config. Let's say server "Lobby" is the default, it will send you there, after that you can connect to another server, for example, "Survival" which is running in a separate server/instance and BungeeCord enables the functionality travel across these different servers without returning to the multiplayer menu screen. (It can be done with special plugins which utilize BungeeCord API to send you to the chosen server on click, you'll most commonly see it as thecompass or a portal)
```

Comment:
```suggestion
No, this can't be used as a normal JAR and requires at least 3 servers/instances, one is BungeeCord (or it's fork) which is called the proxy, players connect to this proxy and are sent to the default server which is set in config. Let's say server "Lobby" is the default, it will send you there, after that you can connect to another server, for example, "Survival" which is running in a separate server/instance and BungeeCord enables the functionality travel across these different servers without returning to the multiplayer menu screen. (It can be done with special plugins which utilize BungeeCord API to send you to the chosen server on click, you'll most commonly see it as a compass or portal).
```